crates/y-sweet-worker/src/config.rs

```diff
@@ -27,9 +42,34 @@ impl From<&Env> for Configuration {
                 })
                 .unwrap_or(45),
         );
+
+        let s3_config = if let (
```

Comment:
I don't love silently failing if some but not all are set. Maybe we make bucket prefix optional, and fail if any of the others are missing (unless they are all missing)?